tests/ZonedDateTimeTest.php

```diff
@@ -408,7 +414,6 @@ public function providerParseInvalidStringThrowsException(): array
             ['2001-02-03T04:05:06[Europe/London]'],
             ['2001-02-03T04:05.789Z[Europe/London]'],
             ['2001-02-03T04:05:06Z[Europe/London'],
-            ['2001-02-03T01:02:03.456+12:34:56'],
```

Comment:
Could you please keep this test for PHP < 8.1.7, as you've done for other tests?